src/InnoSetup/Pages/CmdlinePage.iss

```diff
@@ -147,7 +148,8 @@ begin
       Log('ProcStart failed');
       ExitCode := -2;
     end;
-    while (ExitCode = -1) and not CmdlineInstallCancel do
+    MaxIterations := 42000;
```

Comment:
What this magic number mean?

Reply:
It is a "magic" number, I have decided to go for `The number 42 `and multiply it by 1000.

There is no other special meaning why it has to be this exact number.

Based on the instructions that are executed in the loop and on the ticks of nowadays computers the number has to be in some range to exit the loop within 5 minutes or so, which is the maximal time users would be patient to wait in my opinion.